tsconfig.json

```diff
@@ -3,7 +3,7 @@
     /* Basic Options */
     "target": "ES2017",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-    "lib": ["es2017", "esnext.asynciterable"],                             /* Specify library files to be included in the compilation. */
+    "lib": ["es2017", "esnext.asynciterable", "dom"],                             /* Specify library files to be included in the compilation. */
```

Comment:
This was necessary otherwise apollo-http-link-common's type defs were not compiling:
https://circleci.com/gh/aerogear/apollo-voyager-server/59?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link